index.test.js

```diff
@@ -215,7 +215,7 @@ describe('Deploy to ECS', () => {
                             "value": "world"
                         },
                         {
-                            "name": "",
+                            "name": "test",
```

Comment:
instead of changing the existing test value let's add another env object with a non-empty `name` and then assert:
1) that the non-empty one is retained 
2) that the empty-empty one is still removed

Reply:
Yes, added that.